.github/workflows/build-release.yml

```diff
@@ -86,7 +86,7 @@ jobs:
     - name: Display structure of downloaded files
       run: ls -R
     - name: Create release
-      uses: marvinpinto/action-automatic-releases@latest
```

Comment:
What is the reason for this change? Does it do the same thing?

Reply:
It is a fork of the project of marvinpinto with multiple patches, such as switching to node 16. (GitHUb Action complains about the use of node 12 as it will no longer be supported at some point)

Reply:
The only "problem" I had was the error seen in my second screenshot - maybe I can also resolve it, but afaik it depends on how you release a new version.

Reply:
But it should do essentially the same - docs can be found here: https://github.com/dciborow/action-github-releases#supported-parameters